docker/docker-compose.yml

```diff
@@ -67,6 +67,7 @@ services:
       L1_CHAIN_HTTP: http://l1_chain:8545
       FUEL_GRAPHQL_ENDPOINT: http://fuel_core:4001/graphql
       DEPLOYMENTS_HTTP: http://l1_chain:8081
+      PK_ETH_WALLET: 59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d
```

Comment:
this is the PK of eth wallet, used to deploy the erc20 in `erc20_deployer` docker.

this was added because it was conflicting with the PK used in the `block-commiter`, flakily generating errors